### PR TITLE
we created productTest please review

### PR DESCRIPTION
**Pull request by devikalam**
i made some changes in product test

### CODE REVIEW

File: Com.MySampleFramework/src/test/java/com/ComcastCRMFramework/ContactsTest/ProductTest.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.ComcastCRMFramework.ContactsTest;
+
+public class ProductTest {
+
+	public static void main(String[] args) {
+		System.out.println("product added");
```
Comment (priya260594) [inline Com.MySampleFramework/src/test/java/com/ComcastCRMFramework/ContactsTest/ProductTest.java:6]: Even add some more products
dont use main method

**Review by priya260594 — CHANGES_REQUESTED** [review]
After fixing let me know

**Comment by devikalam** [comment]
i added one more product
i use @Test annotation
please check once

**Comment by priya260594** [comment]
Project got merged.. check it once

**Pull request merged**